README.md

```diff
@@ -64,6 +64,33 @@ class { '::redis':
 }
 ```
 
+### Multiple instances
+
+
+```puppet
```

Comment:
This code would be ideal to copy/paste into an acceptance test and then run a command to test that redis can connect to each of the ports or similar to show that it works.

Reply:
I've modified the original multi-instance acceptance test to use the example code from the README.